crates/subspace-farmer/src/multi_farming.rs

```diff
@@ -20,16 +20,36 @@ pub struct MultiFarming {
     archiving: Archiving,
 }
 
+fn get_plot_sizes(plot_size: u64, max_plot_size: u64) -> Vec<u64> {
```

Comment:
`total_plot_size` would have been better for this, it gets confusing at this point with the variable called like that.